### ISSUE TITLE
Guest mode and add profile still available in drop drown menu after add policies into regedit

### ISSUE THREAD

**Issue by pc-v2** [brave/brave-browser#8907]
## Description 

Guest mode and add profile still available in drop drown menu after add policies into regedit

## Steps to Reproduce

   1. open regedit
      1.a set guest mode and add profile to disable
       

   2. open brave browser drop down menu
   3. it still there

## Actual result:

![image](https://user-images.githubusercontent.com/58063311/77848585-f59c3900-71ef-11ea-9304-a41cd3163519.png)
![image](https://user-images.githubusercontent.com/58063311/77848547-be2d8c80-71ef-11ea-9b75-0ed5db42313a.png)
![image](https://user-images.githubusercontent.com/58063311/77848606-1a90ac00-71f0-11ea-8fea-31b362329a13.png)

## Expected result:
Guest mode and profile manager disabled

## Brave version (brave://version info)

`

Brave | 1.5.115 Chromium: 80.0.3987.149 (Official Build) (64-bit)
-- | --
Revision | 5f4eb224680e5d7dca88504586e9fd951840cac6-refs/branch-heads/3987_137@{#16}
OS | Windows 10 OS Version 2004 (Build 19041.172)
JavaScript | V8 8.0.426.27
Flash | (Disabled)
User Agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.149 Safari/537.36
Command Line | "C:\Program Files (x86)\BraveSoftware\Brave-Browser\Application\brave.exe" --enable-dom-distiller --disable-domain-reliability --no-pings --extension-content-verification=enforce_strict --extensions-install-verification=enforce --sync-url=https://no-thanks.invalid --enable-features=PasswordImport,WebUIDarkMode,SimplifyHttpsIndicator --disable-features=AutofillServerCommunication,AllowPopupsDuringPageUnload,LookalikeUrlNavigationSuggestionsUI,AudioServiceOutOfProcess,SmsReceiver,NotificationTriggers,WebXrGamepadModule,VideoPlaybackQuality,WebXR --flag-switches-begin --load-media-router-component-extension=0 --flag-switches-end --enable-audio-service-sandbox --disable-webrtc-apm-in-audio-service --disable-sync

**Comment by pc-v2** [comment]
is this fixed in latest version?

**Comment by bsclifton** [comment]
@pc-v2 what is the path you're editing? Per https://support.brave.com/hc/en-us/articles/360039248271-Group-Policy it should be:
`HKEY_LOCAL_MACHINE\Software\Policies\BraveSoftware\Brave`

**Comment by pc-v2** [comment]
> @pc-v2 what is the path you're editing? Per https://support.brave.com/hc/en-us/articles/360039248271-Group-Policy it should be:
> `HKEY_LOCAL_MACHINE\Software\Policies\BraveSoftware\Brave`

> @pc-v2 what is the path you're editing? Per https://support.brave.com/hc/en-us/articles/360039248271-Group-Policy it should be:
> `HKEY_LOCAL_MACHINE\Software\Policies\BraveSoftware\Brave`

yes, i use that path already
![image](https://user-images.githubusercontent.com/58063311/79263369-e50ce380-7ebc-11ea-8604-9a3113a8118a.png)

**Comment by bsclifton** [comment]
@pc-v2 curious - what happens if you try to create another profile or open a guest profile?

Definitely seems like we have a bug with those two. Can you confirm that the `Private Window with Tor` option is hidden? And also Incognito disabled as expected?

**Comment by pc-v2** [comment]
> @pc-v2 curious - what happens if you try to create another profile or open a guest profile? Knowing this will help judge severity of the issue (hopefully, it's still blocked but just shows in menu)
> 
> Definitely seems like we have a bug with those two. Can you confirm that the `Private Window with Tor` option is hidden? And also Incognito disabled as expected?

yes, tor and incognito is disabled as expected, but guest window and profile manager still able to access even i disable it (BrowserAddPersonEnabled)

**Comment by vijay2977** [comment]
Any updates on this issue, guys this is an open gaping hole in disabling privacy mode.

**Comment by daweth** [comment]
Any updates? this renders the registries completely useless

**Comment by gelatinbomb** [comment]
It is not only a Brave thing. It appears Vivaldi and Ungoogled Chromium struggles with this too.
This is really bad damn it
In my case the "BrowserGuestModeEnabled" works perfectly on both browsers. On the other side "BrowserAddPersonEnabled" is completely useless.
